app/main.go

```diff
@@ -96,9 +95,53 @@ func main() {
 		log.Fatalf("Invalid flags: %v", err)
 	}
 
-	http.HandleFunc("/", index)
-	http.HandleFunc("/sync", sync)
-	http.HandleFunc("/cleanup", cleanup)
+	config := tsbridge.NewConfig(&tsbridge.ConfigOptions{
+		Filename:                 *metricConfig,
+		MinPointAge:              *minPointAge,
+		CounterResetInterval:     *counterResetInterval,
+		SDLookBackInterval:       *sdLookBackInterval,
+		SDInternalMetricsProject: *sdInternalMetricsProject,
+		UpdateParallelism:        *updateParallelism,
+		UpdateTimeout:            *updateTimeout,
+		EnableStatusPage:         *enableStatusPage,
+		StorageEngine:            *storageEngine,
+		SyncPeriod:               *syncPeriod,
+	})
+
+	h := web.NewHandler(config)
+
+	http.HandleFunc("/", h.Index)
+	http.HandleFunc("/sync", h.Sync)
+	http.HandleFunc("/cleanup", h.Cleanup)
+
+	// Run a cleanup on startup
+	log.Debugf("Performing startup cleanup...")
+	if err := tasks.Cleanup(context.Background(), config); err != nil {
+		log.Errorf("error running the Cleanup() routine: %v", err)
```

Comment:
How do you intend users to detect and respond to this? Perhaps `log.Exitf` would be more appropriate?

Reply:
Agreed, changed to `log.Fatalf()`